client/src/components/showForm/ShowForm.jsx

```diff
@@ -149,7 +150,6 @@ export default function ShowForm() {
             value={date}
             required
             type="date"
-            onChange={(e) => setDate(e.target.value)}
```

Comment:
Why did you delete this? This needs to be undone

Reply:
i put it back in